topics/single-cell/tutorials/alevin-commandline/tutorial.md

```diff
@@ -275,9 +316,27 @@ As you can see, *rowData names* and *colData names* are still empty. Before we a
 
 Some sub-populations of small cells may not be distinguished from empty droplets based purely on counts by barcode. Some libraries produce multiple ‘knees’ (see the [Alevin Galaxy tutorial]({% link topics/single-cell/tutorials/scrna-case_alevin/tutorial.md %}#basic-qc) for multiple sub-populations. The `emptyDrops` method ({% cite Lun2019 %}) has become a popular way of dealing with this. `emptyDrops` still retains barcodes with very high counts, but also adds in barcodes that can be statistically distinguished from the ambient profiles, even if total counts are similar. 
 
+Now, again - install the packages into our R environment: 
+
 ```bash
-library(DropletUtils)               # load the library and required packages
+system("conda install -y bioconductor-DropletUtils", intern=TRUE)
```

Comment:
Would it be better to install those dependencies all together, should safe a bit of time.

Reply:
Thanks for the suggestion! I've already updated it according to your advice.